python/prophet/diagnostics.py

```diff
@@ -121,7 +123,9 @@ def map(self, func, *iterables):
     df = model.history.copy().reset_index(drop=True)
     horizon = pd.Timedelta(horizon)
 
-    predict_columns = ['ds', 'yhat']
+    if predict_columns is None:
+        predict_columns = ['ds', 'yhat']
+        
     if model.uncertainty_samples:
         predict_columns.extend(['yhat_lower', 'yhat_upper'])
```

Comment:
Would we ever want to _not_ output `ds`, `yhat`, `yhat_lower` and `yhat_upper`? I'm wondering if it's better to specify the argument as `extra_output_columns` and do

```python
predict_columns = predict_columns.extend([c for c in extra_output_columns if c not in predict_columns])
```

after line 126.

Reply:
Thank you for the suggestion, added!